rules/falco_rules.yaml

```diff
@@ -3416,6 +3416,21 @@
   priority: CRITICAL 
   tags: [container, mitre_persistence, TA0003]
 
+- rule: Fileless execution via memfd_create
+  desc: 
+    Detect if a binary is executed from memory using the memfd_create technique.
+    This is a well-known defense evasion technique for executing malware on victim machine without storing payload on disk.
+  condition: >
+    spawned_process
+    and proc.is_exe_from_memfd=true
```

Comment:
Any tuning ideas, this likely can be a bit noisy in production?

Reply:
Added some tuning with the classic whitelisting methods. Also, added a description about CVE-2019-5736, which was solved by runc using memfd_create